src/qt/sigmacoincontroldialog.cpp

```diff
@@ -576,27 +542,25 @@ void SigmaCoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
             nChange = nAmount - nPayAmount;
             if (!SigmaCoinControlDialog::fSubtractFeeFromAmount)
                 nChange -= nPayFee;
+        }
 
-            // Never create dust outputs; if we would, just add the dust to the fee.
-            if (nChange > 0 && nChange < MIN_CHANGE)
-            {
-                CTxOut txout(nChange, (CScript)std::vector<unsigned char>(24, 0));
-                if (txout.IsDust(::minRelayTxFee))
-                {
-                    if (SigmaCoinControlDialog::fSubtractFeeFromAmount) // dust-change will be raised until no dust
-                        nChange = txout.GetDustThreshold(::minRelayTxFee);
-                    else
-                    {
-                        nPayFee += nChange;
-                        nChange = 0;
-                    }
-                }
-            }
+        //update bytes
+        CAmount remChange = nChange;
 
-            if (nChange == 0 && !SigmaCoinControlDialog::fSubtractFeeFromAmount)
-                nBytes -= 34;
+        std::vector<int64_t> allDenoms;
+        sigma::GetAllDenoms(allDenoms);
+        for(int64_t denom : allDenoms) {
+            while (remChange >= denom) {
+                nBytes += 44;
```

Comment:
and here, 44 out of nowhere

Reply:
Agree, will add comment. it is the size of each sigma mint.